File: tests/dynamic_fixtures/java/sqli_negative.java
// SQL injection — negative fixture.
// Safe: uses a parameterized query; payload is a bound argument.
// Entry: Entry.login(String)  Cap: SQL_QUERY
// Expected verdict: NotConfirmed

public class Entry {
    public static void login(String username) {
        String template = "SELECT name FROM users WHERE name = ?";
        // Simulate parameterized execution: template is fixed.
        System.out.println("Executing: " + template + " param-len=" + username.length());
    }
}

File: tests/dynamic_fixtures/go/sqli_negative.go
// SQL injection — negative fixture.
// Safe: uses a parameterized query; payload is a bound argument, not concatenated.
// Entry: Login(username string)  Cap: SQL_QUERY
// Expected verdict: NotConfirmed

package entry

import "fmt"

func Login(username string) {
	template := "SELECT name FROM users WHERE name = ?"
	// Simulate parameterized execution: template is fixed.
	fmt.Println("Executing:", template, "with param length:", len(username))
}
